src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -239,13 +239,31 @@ std::vector<ov::Tensor> SubgraphBaseTest::calculate_refs() {
 
     auto functionToProcess = ov::clone_model(*functionRefs);
     //TODO: remove this conversions as soon as function interpreter fully support bf16 and f16
-    static const precisions_array precisions = {
-            { ngraph::element::bf16, ngraph::element::f32 },
+    static const precisions_array f_precisions = {
```

Comment:
```
static precisions_array precisions = {
            { ngraph::element::bf16, ngraph::element::f32 }
    };
    auto convert_added = false;
    for (const auto &param : function->get_parameters()) {
        for (size_t i = 0; i < param->get_output_size(); i++) {
            for (const auto &node : param->get_output_target_inputs(i)) {
                std::shared_ptr<ov::Node> nodePtr = node.get_node()->shared_from_this();
                if (std::dynamic_pointer_cast<ov::op::v0::Convert>(nodePtr)) {
                    convert_added = true;
                    break;
                }
            }
        }
    }
    if (!convert_added) {
        precisions.push_back({ ngraph::element::f16, ngraph::element::f32});
    }
    pass::Manager manager;
    manager.register_pass<ngraph::pass::ConvertPrecision>(precisions);
```

Reply:
Excellent!

Reply:
I removed the static for the vector, it may vary from start to start